source/process/community-guidelines.rst

```diff
@@ -85,11 +85,18 @@ Holding the Meeting
 After the Meeting
 ^^^^^^^^^^^^^^^^^^^^^^^^
 1. Share a summary in all respective channels, including the meeting recording.
+2. Send a survey asking how valuable the meeting was, feedback on additional use cases, and feedback on improving the meeting in future. `See example here <https://community.mattermost.com/core/pl/jfcmekbw738o3f3ykbbhofkg8r>`_.
 
 Tips
 ^^^^^^^^^^^^^^^^^^^^^^^^
 1. Do not use your personal Zoom meeting link for the call.
 2. Note in all announcements that sharing video is optional.
+3. Use `Mattermost` in your name to identify staff from community members on the call.
```

Comment:
Only 1/5 on this one, but after talking to two of the attendees, they said it would have been helpful to know who are Mattermost staff vs community members.